src/counter.rs

```diff
@@ -220,13 +230,18 @@ impl<P: Atomic> Clone for GenericLocalCounter<P> {
     }
 }
 
+/// The underlying implementation for [`LocalCounterVec`](::local::LocalCounterVec)
+/// and [`LocalIntCounterVec`](::local::LocalIntCounterVec).
 pub struct GenericLocalCounterVec<P: Atomic> {
     vec: GenericCounterVec<P>,
     local: HashMap<u64, GenericLocalCounter<P>>,
 }
 
+/// An unsync thread local copy of [`CounterVec`](::CounterVec).
```

Comment:
I think `thead local` is a little bit misleading.

Reply:
agree. I think `unsync` is enough?

Reply:
Yes, I think `unsync` is enough too.